src/LightsOff.cpp

```diff
@@ -0,0 +1,232 @@
+// "Lights Off" module for VCV Rack
+//
+// Copyright (C) 2020 Benjamin Dill
+// All rights reserved.
```

Comment:
Is this license compatible with the BSD-3-Clause license of the project as a whole?

Reply:
Sorry, the first line got dropped accidently. It is the BSD-3-Clause licence. Fixed in 
2cd3168ad1bfcdc6cd5299e1376fbaf09e9f8eed.